### PR TITLE
feat: support sepolia

### PR DESCRIPTION
**Pull request by TTNguyenDev**
image
#135

### CODE REVIEW

**Comment by TTNguyenDev** [comment]
Please check this PR. #146 closed

**Comment by TTNguyenDev** [comment]
@aahna-ashina I'm wondering what is the cohort address for sepolia?

File: packages/web-app/lib/connectors.tsx [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
 import { publicProvider } from "wagmi/providers/public";
 
 type FallbackProviderConfig = Omit<providers.FallbackProviderConfig, "provider">;
 
 const customAlchemyProvider = ({ priority, stallTimeout, weight }: FallbackProviderConfig) => {
 	return (chain: Chain) => {
-		const apiKey = process.env.NEXT_PUBLIC_ALCHEMY_API_KEY; //mainnet
+		let apiKey = process.env.NEXT_PUBLIC_ALCHEMY_API_KEY;
+		if (chain?.id == 11155111) {
+			apiKey = process.env.NEXT_PUBLIC_ALCHEMY_API_KEY_SEPOLIA;
```
Comment (aahna-ashina) [inline packages/web-app/lib/connectors.tsx:13]:
```suggestion
			apiKey = process.env.NEXT_PUBLIC_ALCHEMY_API_KEY;
```

File: packages/web-app/lib/constants/sepolia.tsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { BigNumber } from "ethers";
+
+export const permit2Address = "0x000000000022D473030F116dDEE9F6B43aC78BA3";
+export const NATION = "0x23Ca3002706b71a440860E3cf8ff64679A00C9d7";
+export const frameworkAddress = "0xD96aA6e2568f4e9632D2A5234Bb8410ca7609a27";
+export const arbitratorAddress = "0xBe67cEdCD1FE38aac8a5781A51250FDeFB344E6C";
+export const cohortAddress = "0xfbb66bc799308435ed2a0e0c0ac3ad1d46749b7b";
+export const safeTxServiceUrl = "https://safe-transaction-sepolia.safe.global";
+export const subgraphURI = process.env.NEXT_PUBLIC_GRAPH_API_URL_SEPOLIA;
```
Comment (aahna-ashina) [inline packages/web-app/lib/constants/sepolia.tsx:9]:
```suggestion
export const subgraphURI = process.env.NEXT_PUBLIC_GRAPH_API_URL;
```

**Review by aahna-ashina — DISMISSED** [review]
🙌

**Comment by aahna-ashina** [comment]
> @aahna-ashina I'm wondering what is the cohort address for sepolia?

Sorry, I don't know what the `cohortAdress` is for.  Maybe there is some documentation mentioning what "cohort" means here?

**Comment by TTNguyenDev** [comment]
@aahna-ashina this is cohort address on mainnet `export const cohortAddress = "0xee8F0B89983aF83598E6Fb503B9f6e5eaf23b243";`

File: packages/web-app/lib/constants/sepolia.tsx (hunk 2 above)
Comment (TTNguyenDev) [reply packages/web-app/lib/constants/sepolia.tsx:9]: Does this mean that https://agreements.nation3.org/ supports only the Ethereum mainnet, and https://sepolia.agreements.nation3.org/ supports only the Sepolia testnet, correct?

Comment (TTNguyenDev) [reply packages/web-app/lib/constants/sepolia.tsx:9]: we don't support network switching?

**Comment by TTNguyenDev** [comment]
https://etherscan.io/address/0xee8F0B89983aF83598E6Fb503B9f6e5eaf23b243#code

File: packages/web-app/lib/constants/sepolia.tsx (hunk 2 above)
Comment (aahna-ashina) [reply packages/web-app/lib/constants/sepolia.tsx:9]: > we don't support network switching?

We can still support it, by redirecting to the other URLs.  If a user opens https://agreements.nation3.org and selects Sepolia in the chain selection drop-down, we can redirect to https://agreements-sepolia.nation3.org

**Comment by aahna-ashina** [comment]
> @aahna-ashina this is cohort address on mainnet `export const cohortAddress = "0xee8F0B89983aF83598E6Fb503B9f6e5eaf23b243";`

Okay, so a _cohort_ will always be a Gnosis safe?  I just don't understand the meaning of "cohort" in this context.  The variable should have a different name so that everyone can understand what it's for.  Can we rename it to something more meaningful?

**Comment by TTNguyenDev** [comment]
Will the Sepolia cohort's address be the same as Mainnet's, @aahna-ashina?

**Review by aahna-ashina — APPROVED** [review]

**Pull request merged**